Misc. typos

luzpaz opened the pull request:
Found via `codespell -q 3 -I ../godot-word-whitelist.txt --skip="./thirdparty,*.po"`

akien-mga commented:
Your branch is 3000 commits behind the current master branch, so sadly there are conflicts (and potentially other typos that your commit missed).

akien-mga commented on modules/csg/csg.cpp:
```diff
@@ -573,7 +573,7 @@ void CSGBrushOperation::_collision_callback(const CSGBrush *A, int p_face_a, Map
 	}
 
-	//if we are still here, it means they most likely intersect, so create BuildPolys if they dont existy
+	//if we are still here, it means they most likely intersect, so create BuildPolys if they don't existy
```
There's another typo here which was missed: "existy".

akien-mga commented on scene/gui/graph_edit.cpp:
```diff
@@ -1304,7 +1304,7 @@ GraphEdit::GraphEdit() {
 	connections_layer->connect("draw", this, "_connections_layer_draw");
 	connections_layer->set_name("CLAYER");
-	connections_layer->set_disable_visibility_clip(true); // so it can draw freely and be offseted
+	connections_layer->set_disable_visibility_clip(true); // so it can draw freely and be offsetted
```
Should be "offset" (same past participle as "set").

akien-mga commented on platform/android/android_native_app_glue.h:
```diff
@@ -50,7 +50,7 @@ extern "C" {
  *
  * 2/ android_main() receives a pointer to a valid "android_app" structure
  *    that contains references to other important objects, e.g. the
- *    ANativeActivity obejct instance the application is running in.
```
This file (`platform/android/android_native_app_glue.h`) and `platform/android/cpu-features.c` come directly from Google, so it's better not to modify them in our repo. I should move them to `thirdparty/` eventually.

akien-mga commented:
I made a couple comments but otherwise it should be ready to squash and merge.

luzpaz commented:
Made the requested changes in:
6afa589
9b8cd77
Shall I rebase and squash?

akien-mga commented on platform/javascript/os_javascript.cpp:
```diff
@@ -708,7 +708,7 @@ Error OS_JavaScript::initialize(const VideoMode &p_desired, int p_video_driver,
 
 	video_mode = p_desired;
-	// Can't fulfill fullscreen request during start-up due to browser security.
+	// Can't fulfil fullscreen request during start-up due to browser security.
```
This one could be kept, `os_javascript.cpp` is our own. It's `platform/android/cpu-features.c` that must be reverted.

luzpaz replied:
2b29cff

akien-mga commented on platform/android/cpu-features.c:
```diff
@@ -187,7 +187,7 @@ read_file(const char*  pathname, char*  buffer, size_t  buffsize)
 }
 
-/* Extract the content of a the first occurence of a given field in
+/* Extract the content of a the first occurrence of a given field in
```
The changes to this file still need to be undone.

luzpaz commented:
Should be good to go, AFAICT.

luzpaz committed:
Misc. typos

Found via `codespell -q 3 -I ../godot-word-whitelist.txt --skip="./thirdparty,*.po"`

akien-mga commented:
Thanks a lot!
